types/params.go

```diff
@@ -7,40 +7,68 @@ import (
 	cmtypes "github.com/cometbft/cometbft/types"
 )
 
+var (
+	// ErrMaxBytesCannotBeZero indicates that the block's MaxBytes cannot be 0.
+	ErrMaxBytesCannotBeZero = errors.New("block.MaxBytes cannot be 0")
+
+	// ErrMaxBytesInvalid indicates that the block's MaxBytes must be either -1 or greater than 0.
+	ErrMaxBytesInvalid = errors.New("block.MaxBytes must be -1 or greater than 0")
+
+	// ErrMaxBytesTooBig indicates that the block's MaxBytes exceeds the maximum allowed size.
+	ErrMaxBytesTooBig = errors.New("block.MaxBytes is too big")
+
+	// ErrMaxGasInvalid indicates that the block's MaxGas must be greater than or equal to -1.
+	ErrMaxGasInvalid = errors.New("block.MaxGas must be greater or equal to -1")
```

Comment:
really MaxGas can be zero? Wouldn't that prevent txns from being submitted? Or is it possible for gasless txns to be included if there isn't a lot of competition?

Reply:
Yes, seems like it: https://docs.cometbft.com/v0.34/spec/abci/apps#blockparamsmaxgas, left for the app to enforce.